src/components/layout/Navbar.test.tsx

```diff
@@ -127,6 +127,18 @@ describe(`Navbar`, () => {
         await wrapper.getByPlaceholderText('Search for videos'),
       ).toBeVisible();
     });
+
```

Comment:
When do we want to hide search?

Reply:
On the homepage the navbar is rendered without search as it has been moved to the main page content